common/Helpers/ManagementInfrastructureHelper.cs

```diff
@@ -36,7 +42,19 @@ public static FeatureAvailabilityKind IsWindowsFeatureAvailable(string featureNa
                     var featureAvailability = GetAvailabilityKindFromState(installState);
 
                     _log.Information($"Found feature: '{featureName}' with enablement state: '{featureAvailability}'");
-                    return featureAvailability;
+
+                    // Most optional features do not have a description, so we provide one for known features
+                    var description = featureInstance.CimInstanceProperties["Description"]?.Value as string;
+                    if (string.IsNullOrEmpty(description) && WindowsOptionalFeatures.FeatureDescriptions.TryGetValue(featureName, out var featureDescription))
+                    {
+                        description = featureDescription;
+                    }
+
+                    return new FeatureInfo(
+                        featureName,
+                        featureInstance.CimInstanceProperties["Caption"]?.Value as string ?? featureName,
+                        description ?? string.Empty,
```

Comment:
NIT: Move `?? string.empty` to line 47 where description is declared.  One less assignment statement.